docs/zh/reference/ip_tips.md

```diff
@@ -0,0 +1,56 @@
+# IP Tips
+
+## 物理环境 IP
+跨主机访问OpenMLDB服务，需要将OpenMLDB配置中的endpoint配置`127.0.0.1`改为`0.0.0.0`或公网IP，再启动OpenMLDB服务。请保证端口没有被防火墙阻挡。
+```{attention}
+单机版中，不只是需要改endpoint，nameserver的配置中还有tablet ip `--tablet=`，此处也需要修改。
+```
+
+## Docker IP
+
+希望从容器的外部（无论是同一主机还是跨主机）访问容器内，请先
+更改endpoint`127.0.0.1`为 `0.0.0.0`（单机版中`tablet`配置项也需要更改），以避免不必要的麻烦。
+
+### 容器外部访问（同一主机）
+在同一主机中，想要从**容器的外部**（物理机或者是其他容器）访问**容器内**启动的OpenMLDB服务端，可以直接使用bridge的方式连接，也可以暴露端口，还可以直接使用host网络模式。
+
+```{caution}
+Docker Desktop for Mac无法支持从物理机访问容器（以下任何模式都不能），参考[i-cannot-ping-my-containers](https://docs.docker.com/desktop/mac/networking/#i-cannot-ping-my-containers)。
+
+但macOS中，可以从容器内访问其他容器。
+```
+
+#### bridge连接
+bridge连接不需要更改docker run命令，只需要查询一下bridge ip。
+```
+docker network inspect bridge
+```
+查看“Containers”字段，可以看到每个容器绑定的ip，客户端使用该ip就可以进行访问。
+
+例如，启动容器并运行OpenMLDB单机版后，inspect结果为`172.17.0.2`，那么CLI连接可以使用：
+```
+../openmldb/bin/openmldb --host 172.17.0.2 --port 6527
+```
+
+#### 暴露端口
+在启动容器时通过 `-p` 暴露端口，客户端可以使用本机ip地址或回环地址进行访问。
+
+单机版需要暴露三个组件（nameserver，tabletserver，apiserver）的端口：
+```
+docker run -p 6527:6527 -p 9921:9921 -p 8080:8080 -it 4pdosc/openmldb:0.4.4 bash
+```
+
+集群版需要暴露zk端口与所有组件的端口：
+```
+docker run -p 2181:2181 -p 7527:7527 -p 10921:10921 -p 10922:10922 -p 8080:8080 -p 9902:9902 -it 4pdosc/openmldb:0.4.4 bash
```

Comment:
we should manually expose port in Dockerfile later

Reply:
👍